client/src/components/Card/Card.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Card.css'
+import Icons from '../Icons/FAIcon'
+
+let types = ['heart', 'calendar', 'dress']
+let useIcon = null
+
+const Card = (props) => {
+	if (types.includes(props.type)) {
+		useIcon = (props.icon)
+	}
```

Comment:
This is clever! You can simplify this a lot by just passing dow the icon as a prop